### PR TITLE
Metadata less locks

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: src/node/metadata/elder_stores.rs [hunk 1]
```diff
@@ -36,38 +37,54 @@ impl ElderStores {
...
         msg_id: MessageId,
         requester: PublicKey,
         origin: EndUser,
     ) -> Result<NodeDuty> {
         match &query {
-            DataQuery::Blob(read) => self.chunk_records.read(read, msg_id, origin).await,
-            DataQuery::Map(read) => self.map_storage.read(read, msg_id, requester, origin).await,
+            DataQuery::Blob(read) => {
+                self.chunk_records
+                    .write()
```
Comment (dirvine) [inline src/node/metadata/elder_stores.rs:57]: Should this not be `read`

Comment (joshuef) [reply src/node/metadata/elder_stores.rs:57]: no as we (currently) write the liveness checks inside here. 

Something to do (was leaving it for another PR) is to return another `NodeDuty` for writing read receipts for adults so we can have that compeltely separate from chunk storage.

File: src/node/metadata/elder_stores.rs [hunk 2]
```diff
@@ -78,40 +95,53 @@ impl ElderStores {
             DataCmd::Blob(write) => {
                 info!("Writing Blob");
                 self.chunk_records
+                    .write()
```
Comment (lionel1704) [inline src/node/metadata/elder_stores.rs:98]: I think this can be `read()` since there is nothing being written here.

Comment (lionel1704) [reply src/node/metadata/elder_stores.rs:98]: We only send the data to Adults. (no liveliness checks too)

Comment (joshuef) [reply src/node/metadata/elder_stores.rs:98]: not yet, that'll be possible with the https://github.com/maidsafe/safe_network/pull/121 in i think

**Review by dirvine — APPROVED** [review]

**Comment by joshuef** [comment]
Closing as this should get atop transfer removal for simplicity (or wont be needed there)

**Pull request closed without merging**